tests/testthat/test-profile_emissions_upstream.R

```diff
@@ -52,6 +52,36 @@ test_that("the output at product level has columns matching isic and sector", {
   expect_true(any(matches_name(product, "sector")))
 })
 
+test_that("the output at product and company level has columns `co2e_lower` and `co2e_upper`", {
+  local_options(readr.show_col_types = FALSE)
```

Comment:
This is no longer necessary. See https://github.com/2DegreesInvesting/tiltIndicatorAfter/issues/133